sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyServiceClient.java

```diff
@@ -125,15 +125,15 @@ JsonWebKey transformSecretKey(SecretKey secretKey) throws JsonProcessingExceptio
         return mapper.readValue(jsonString, JsonWebKey.class);
     }
 
-    Mono<EncryptResult> encrypt(EncryptOptions encryptOptions, Context context) {
-        Objects.requireNonNull(encryptOptions, "'encryptOptions' cannot be null.");
+    Mono<EncryptResult> encrypt(EncryptParameters encryptParameters, Context context) {
+        Objects.requireNonNull(encryptParameters, "'encryptOptions' cannot be null.");
```

Comment:
Fix all the error messages.
```suggestion
        Objects.requireNonNull(encryptParameters, "'encryptParameters' cannot be null.");
```